products/compute/api.yaml

```diff
@@ -1058,21 +1058,22 @@ objects:
           character, which cannot be a dash.
         required: true
       - !ruby/object:Api::Type::NameValues
-        name: 'labels'
-        description: |
-          Labels to apply to this address.  A list of key->value pairs.
-        key_type: Api::Type::String
-        value_type: Api::Type::String
-        update_verb: :POST
-        update_url: 'projects/{{project}}/global/addresses/{{name}}/setLabels'
-        min_version: beta
-      - !ruby/object:Api::Type::Fingerprint
-        name: 'labelFingerprint'
-        description: |
-          The fingerprint used for optimistic locking of this resource.  Used
-          internally during updates.
-        update_url: 'projects/{{project}}/global/addresses/{{name}}/setLabels'
+        name: 'labels'	
```

Comment:
why are we adding empty space to the end of every line?

Reply:
Typo! I'll fix this and then get some YAML linting happening.